src/views/Alteration.vue

```diff
@@ -245,6 +272,14 @@ export default class Alteration extends Mixins(CommonMixin, LegalApiMixin, Filin
         priority: false
       })
 
+      // update the current fees for the Filing
+      this.setCurrentFees(await this.fetchFilingFees(
+        FilingCodes.ALTERATION, this.getEntityType, this.getEffectiveDateTime.isFutureEffective
+      ) || {})
+
+      // fetches the fee prices to display in the text
+      this.setFeePrices(await this.fetchFilingFees(FilingCodes.ALTERATION, this.getEntityType, true) || {})
```

Comment:
I'm not sure that this works as you expect...
I believe `fetchFilingFees` can throw an error, which you don't catch here, so it will bubble up to the main method and cause alteration loading to fail.
When an awaited method throws an error, it does not return falsy (which is what this code seems to expect).
You can catch the error like this (note the `catch`, which returns {}):
```
const filingFees = await this.fetchFilingFees(FilingCodes.ALTERATION, this.getEntityType, true).catch(() => {})
this.setFeePrices(filingFees)
```

Reply:
ok. I see.. you're right. Thanks!

Reply:
PS You can test this by blocking the endpoint in Chrome dev tools. Or you can edit fetchFilingFees to try to call an endpoint that doesn't exist (eg /andre). And then see what the UI does...

Reply:
Yes, I did the test with Chrome Dev Tools and I could check you're really right. I didn't know we could do that in dev tools. Thanks for the tip!